docs/tutorials/Raster-data-Lambert93.md

```diff
@@ -20,20 +20,23 @@ The webpage we want to display data on should be structured as follows :
     </head>
     <body>
         <div id="viewerDiv"></div>
-        <script src="js/itowns.js"></script>
+        <script src="../dist/itowns.js"></script>
         <script type="text/javascript">
-            // Our code goes here
+            // Tutorial code should go here
         </script>
     </body>
 </html>
 ```
 
 What we are doing here is fairly simple :
-- we create a container `<div id="viewerDiv"></div>` that shall serve as a support for the data we wish to display ;
+- we create a container `<div id="viewerDiv"></div>` that will contain the 3D view ;
 - we define the layout of this container within our webpage ;
-- we import iTowns framework (which in our case is imported within a `js/` repository).
+- we import iTowns framework (which in our case is imported from the `dist/` repository).
 
-We shall then add javascript to use iTowns to display data within a `<script>` markup following iTowns import.
+We will then add javascript to use iTowns to display data within a `<script>` markup following iTowns import.
+
+To work as is, this web page should be placed in the `examples/` folder of itowns but you can put it anywhere else as 
```

Comment:
Perhaps we could add a link to the code in github when mentioning itowns ?

Reply:
Do you mean a link to the `examples/` folder ?

Reply:
No, I meant replacing `itowns` by `[itowns](https://github.com/iTowns/itowns)`, so that anyone who reads the tutorial without having visited the github before gets an easy access to clone the code.

Reply:
ok, done